staging/src/k8s.io/apimachinery/pkg/api/apitesting/roundtrip/roundtrip.go

```diff
@@ -24,8 +24,6 @@ import (
 	"strings"
 	"testing"
 
-	//nolint:staticcheck //iccheck // SA1019 Keep using deprecated module; it still seems to be maintained and the api of the recommended replacement differs
```

Comment:
hrmm... I'm trying to remember the context for these comments and what incompatibility it was referencing

Reply:
This was added at https://github.com/kubernetes/kubernetes/pull/102629 . I think he just want to fix the warning and the new module does not provide direct replacement for `DebugPrint`.
Now I've tested that `DebugPrint` does not print anything useful in our case, I think it is safe to just remove it.